sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/AmqpConnection.java

```diff
@@ -57,6 +57,17 @@
      */
     Mono<AmqpSession> createSession(String sessionName);
 
+    /**
+     * Creates a new session with the given session name and set it up for distributed transaction if required.
+     *
+     * @param sessionName Name of the session to be created.
+     * @param distributedTransactionsSupport If this session should support distributed transactions across entities.
+     * @return The AMQP session that was created.
+     *
+     * @see <a href="http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-transactions-v1.0-os.html#section-coordination">Distributed Transactions</a>
+     */
+    Mono<AmqpSession> createSession(String sessionName, boolean distributedTransactionsSupport);
```

Comment:
Adding a new method to the interface is a breaking change.

Reply:
Removed the need for rev API changes.